extension/pybindings/pybindings.cpp

```diff
@@ -975,6 +979,18 @@ py::list get_operator_names() {
   return res;
 }
 
+py::list get_backend_names() {
```

Comment:
get_registered_backends

Reply:
updated to `get_registered_backend_names`